composer/utils/checkpoint.py

```diff
@@ -625,18 +626,18 @@ def download_checkpoint(path: str,
                     # or could not be downloaded
                     raise RuntimeError(f'Checkpoint {path} does not exist')
         elif checkpoint_is_sharded:
-            # if the checkpoint is sharded, then every rank needs to download its own checkpoint
+            # If the checkpoint is sharded, then every rank needs to download its own checkpoint
+            path = _format_path_with_current_rank(path)
```

Comment:
just checking that I'm not misreading, this section is just a refactor with no logical changes, yeah?

Reply:
Yep